### PR TITLE
Create db_setup.md

### PR DESCRIPTION
**Pull request by Dezash**

### CODE REVIEW

**Review by wizacass — COMMENTED** [review]
Mb we could try doing a script that would take the parameters from some .ENV file or just a regular file and run all the commands?

**Comment by Dezash** [comment]
@wizacass I don't see a reason for that since environment variables will overwrite app-secrets. Using both would be redundant.

File: db_setup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+`dotnet users-secrets init`
+
+`dotnet users-secrets set "Server" "YourServerName"`
+
+`dotnet users-secrets set "Database" "YourDatabaseName"`
+
+`dotnet users-secrets set "Uid" "YourUsername"`
+
+`dotnet users-secrets set "DbPassword" "YourPassword"`
```
Comment (wizacass) [inline db_setup.md]: The command is `dotnet user-secrets`. Please change accordingly.

**Review by wizacass — CHANGES_REQUESTED** [review]

File: db_setup.md (hunk 1 above)
Comment (Dezash) [reply db_setup.md]: Done

**Review by lukasrakauskas — APPROVED** [review]

**Review by wizacass — APPROVED** [review]

**Pull request merged**